### PR TITLE
Add docs for Azure 2.2.x plugin version

### PR DESCRIPTION
**Pull request by chris1984**
@upadhyeammit 

Amit here are the docs for 2.2.x

### CODE REVIEW

File: plugins/foreman_azure/2.2.x/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+* Once the compute resource with Azure Resource Manager provider is created, next step is Image creation. AzureRm now supports provisioning with marketplace, custom and shared gallery images.
+
+* The only point to remember is while specifying the Image name it should be prefixed with the type of image. For marketplace, custom and gallery images, prefixes should be - _marketplace://_, _custom://_ and _gallery://_ respectively.
+
+* You are now ready to create the Host. You have to select your Azure Resource Manger compute resource on the Hosts page and the relevant fields shall appear on the Virtual Machine tab.
+
+* You can now fill in your resource group, size of the VM you want to provision, username and password or sshkey with which the user can login to the provisioned machine.
+
+* Now, you can also select additional disks for the host. Note that the maximum number of these disks depends on the VM Size selected. Please refer to MSFT documentation for more details.
```
Comment (upadhyeammit) [inline plugins/foreman_azure/2.2.x/index.md]: Is it possible to add link of MSFT documentation?

File: plugins/foreman_azure/2.2.x/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+* In addition to that, you can provide any custom script or command to be executed during the VM creation process.
+
+* In the Operating Systems tab, you must specify the Architecture, OS and the relevant Image that you have created under the compute resource. Although, setting up Root Password is not mandatory.
+
+* Last specification is for the Network Interface. You should select a domain defined under Foreman and then select Azure subnet and Public or Private IP format. Now, you can also select multiple interfaces for your host. The maximum number of these interfaces depends on the VM Size selected. Please refer to MSFT documentation for more details.
```
Comment (upadhyeammit) [inline plugins/foreman_azure/2.2.x/index.md]: Ditto, request to add MSFT documentation.

**Review by upadhyeammit — CHANGES_REQUESTED** [review]

File: plugins/foreman_azure/2.2.x/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+version: 2.2.x
+# uncomment for older versions than stable or nightly
+#robots: noindex
+---
+
+# 1. Basics
+
+Foreman AzureRm is the plugin for Azure v2 i.e. Azure Resource Manager API. It allows you to provision and manage your hosts in Azure Resource Manager cloud environment. Users do have restricted access within the Azure subscription (such that only they can create hosts) based on Resource Groups in AzureRm.
+
+This plugin 2.2.x series has no dependency of foreman_azure 1.x as they refer to different Azure APIs. This 2.2.x should be compatible with Foreman 2.0 onwards, but it is recommended to use latest Foreman version.
```
Comment (melcorr) [inline plugins/foreman_azure/2.2.x/index.md]:
```suggestion
This plugin 2.2.x series has no dependency on foreman_azure 1.x as they refer to different Azure APIs. This 2.2.x should be compatible with Foreman 2.0 onwards, but it is recommended to use latest Foreman version.
```

File: plugins/foreman_azure/2.2.x/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    $ gem 'foreman_azure_rm'
+
+Then run `bundle install` from the same directory
+
+# 3. Configuration
+
+Please refer to our documentation [Github](https://github.com/theforeman/foreman_azure_rm#foreman-azurerm-plugin)
+
+# 4. Usage
+* Creation of the Compute Resource requires using of four credentials: Client ID, Client Secret, Subscription ID and Tenant ID.
```
Comment (melcorr) [inline plugins/foreman_azure/2.2.x/index.md]:
```suggestion
* Creation of the Compute Resource requires using four credentials: Client ID, Client Secret, Subscription ID and Tenant ID.
```

**Review by melcorr — COMMENTED** [review]
Tiny nits

**Comment by melcorr** [comment]
I'm going to link from this PR to a new issue in the new docs, and whenever I get a chance, I'll update there also: https://docs.theforeman.org/nightly/Provisioning_Guide/index-foreman-el.html#provisioning-cloud-instances-azure_provisioning

File: plugins/foreman_azure/2.2.x/index.md (hunk 3 above)
Comment (chris1984) [reply plugins/foreman_azure/2.2.x/index.md]: Fixed.

File: plugins/foreman_azure/2.2.x/index.md (hunk 4 above)
Comment (chris1984) [reply plugins/foreman_azure/2.2.x/index.md]: Fixed.

File: plugins/foreman_azure/2.2.x/index.md (hunk 2 above)
Comment (chris1984) [reply plugins/foreman_azure/2.2.x/index.md]: Fixed.

File: plugins/foreman_azure/2.2.x/index.md (hunk 1 above)
Comment (chris1984) [reply plugins/foreman_azure/2.2.x/index.md]: Fixed.

**Comment by chris1984** [comment]
@melcorr @upadhyeammit updated, let me know how that looks.

**Review by upadhyeammit — APPROVED** [review]
LGTM!

**Pull request merged**